src/routes/index.js

```diff
@@ -1,6 +1,20 @@
 module.exports = function(express) {
+  var express = require('express');
```

Comment:
Express is already established in the line before this, so this line is redundant. Please remove.